### ISSUE TITLE
What is the parameter in case 

### ISSUE THREAD

**Issue by kishor83** [mbuesch/pyprofibus#13]
; Only for type=serial:
; The PHY device name/path.
; Can be a device like /dev/ttyS0 or /dev/ttyAMA0
**dev=/dev/ttyS0**

what is this setting we should give in windows machine ? how and where can we find  this?

**Comment by mbuesch** [comment]
The CP5711 is not supported by pyprofibus.

You could use a PCI(e) card with serial port + RS485 converter and use COM1, COM2, etc instead.

**Comment by kishor83** [comment]
@mbuesch That was quick info, Thank you for your initiataive bringing PB to python world

did you mean following when you say PCI(e) card ? a CP 5611 ?
6GK1561-1AA01
Communication processor CP 5611 A2
Communications processor CP 5611 A2 PCI card (32 bit; 3.3/5 V; 33/66MHz) to Connection of a PG or PC with PCI Bus to PROFIBUS or MPI

**Comment by mbuesch** [comment]
No, I'm talking about a standard PC serial port. That can be built into the PC (less common these days) or via cheap PCI card.
Such a RS232 port can be connected to Profibus via RS232->RS485 converter dongle.

It's *not* recommended to use USB->RS232 dongles.

None of the Siemens CP devices is supported by pyprofibus.
